android/app/src/main/java/com/microsoft/codepush/common/datacontracts/CodePushReportStatusResult.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.codepush.common.datacontracts;
+
+import com.microsoft.codepush.common.exceptions.CodePushReportStatusException;
+
+/**
+ * Represents the result of sending status report to server.
+ */
+public class CodePushReportStatusResult {
+
+    /**
+     * An exception that has occurred when sending the report, <code>null</code> if the report has been sent successfully.
+     */
+    private CodePushReportStatusException codePushReportStatusException;
+
+    /**
+     * The result string from server.
+     */
+    private String result;
```

Comment:
`mResult`

Reply:
@MatkovIvan In data contracts classes we do not use this convention as it is in AppCenter